src/test/resources/features/US001.feature

```diff
@@ -1,4 +1,5 @@
 Feature: deneme
```

Comment:
test gecmedi, hata verdi
Given user goes to "https://test.hypnotes.net/" -> org.openqa.selenium.WebDriverException: unknown error: net::ERR_NAME_NOT_RESOLVED